### PR TITLE
fix console errors

### PR DESCRIPTION
**Pull request by itshady**
# Description

Noticed console errors when running the dev build. Just minor issues that won't affect the code, but good to keep clean.
All children need unique keys, so it is sufficient to pass in the names of the events as keys.

image

## Type of change

Please check option which applies.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**: Just dev up and check the website, nothings changed, just minor console error fixes
* OS: Windows
* Browser: Chrome

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have performed successful dev and production local builds
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by adarsh-swe** [comment]
@itshady are the changes made to navbar.js and seo.js required for the events section bug fix?

File: src/components/Events/Events.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ function Events() {
             {EventData.filter(edge => {
               return stringToDate(edge.node.date) > new Date();
             }).map((edge, i) => {
-              return i < maxEvents && <EventItem {...edge.node} />;
+              return i < maxEvents && <EventItem {...edge.node} key={edge.node.title} />;
```
Comment (adarsh-swe) [inline src/components/Events/Events.js:67]: Are there any other unique identifiers that can be used?

**Review by adarsh-swe — CHANGES_REQUESTED** [review]
The events section fix looks good to merge but if there is another unique identifier we can use, lets change the key incase any events have the same title (eg. leetcode nights might have the same title but different name). One suggestion could be to just use the counter 'i' from the .map() callback function as the key.

File: src/components/Events/Events.js (hunk 1 above)
Comment (itshady) [reply src/components/Events/Events.js:67]: using i is much smarter! Just changed it

**Comment by itshady** [comment]
> @itshady are the changes made to navbar.js and seo.js required for the events section bug fix?

@adarsh-swe Not for the events section bug fix. These are other bug fixes, but the events one seems to be working on this branch. I assume the production branch (not sure which) isn't up to date

**Review by adarsh-swe — APPROVED** [review]

**Pull request merged**